datastream-kafka/src/main/java/com/linkedin/datastream/kafka/KafkaProducerWrapper.java

```diff
@@ -296,7 +296,6 @@ void send(DatastreamTask task, ProducerRecord<K, V> producerRecord, Callback onC
           Thread.sleep(_sendFailureRetryWaitTimeMs);
         }
       } catch (Exception e) {
-        _log.error(String.format("Send failed for partition %d with an exception", producerRecord.partition()), e);
```

Comment:
Is there any other log for this code path that tells us the partition and exception? Just want to make sure we'll have sufficient logs to debug issues if this occurs.